### PR TITLE
[PF-1837] Install cromwell in post-startup.sh.

### PR DESCRIPTION
**Pull request by yuhuyoyo**
https://cromwell.readthedocs.io/en/stable/tutorials/FiveMinuteIntro/

Create a new cromwell directory and download the 81 (latest) cromwell jar in the directory.

### CODE REVIEW

File: service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh [hunk 1]
```diff
@@ -55,6 +56,13 @@ fi
 sudo -u "${JUPYTER_USER}" sh -c "curl -s https://get.nextflow.io | bash"
 sudo mv nextflow /usr/bin/nextflow
 
+# Install cromwell
+readonly CROMWELL_LATEST_VERSION="81"
+sudo -u "${JUPYTER_USER}" sh -c "mkdir -p /home/${JUPYTER_USER}/cromwell"
+cd /home/${JUPYTER_USER}/cromwell
+sudo -u "${JUPYTER_USER}" sh -c "curl -LO https://github.com/broadinstitute/cromwell/releases/download/${CROMWELL_LATEST_VERSION}/cromwell-${CROMWELL_LATEST_VERSION}.jar | bash"
```
Comment (mikenev) [inline service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh:63]: curious why this pipes to bash at the end. Do we execute something in the .jar file?

Comment (yuhuyoyo) [reply service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh]: good question. it was just copied from above. i removed it.

**Review by mikenev — APPROVED** [review]

**Review by zloery — APPROVED** [review]

**Comment by yuhuyoyo** [comment]
integration test passing in verily: https://github.com/verily-src/terra-service-ws-manager/actions/runs/2713869585
https://github.com/verily-src/terra-service-ws-manager/pull/162

**Comment by jmczerk** [comment]
Merging based on test results in Verily environment and known issue ID-161.

**Pull request merged**